pkg/controller/acmechallenges/controller.go

```diff
@@ -207,15 +207,15 @@ func (c *controller) ProcessItem(ctx context.Context, key types.NamespacedName)
 	namespace, name := key.Namespace, key.Name
 
 	ch, err := c.challengeLister.Challenges(namespace).Get(name)
-
-	if err != nil {
-		if k8sErrors.IsNotFound(err) {
-			log.Error(err, "challenge in work queue no longer exists")
```

Comment:
These logs are not useful in my opinion.
AFAIK, it is very normal for items in the workqueue to have been deleted before they start being reconciled, causing this error.